src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &-star {
+    height: 12px;
+    width: 12px;
+    background-size: cover;
+
+    margin: 2px;
+    margin-right: 6px;
```

Comment:
Using both 'margin' and 'margin-right' together can be redundant. Consider using 'margin' shorthand to define all sides at once if possible.